elki/src/main/java/ch/ethz/globis/pht/pre/ColumnType.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2011-2015 ETH Zurich. All Rights Reserved.
+ *
+ * This software is the proprietary information of ETH Zurich.
+ * Use is subject to license terms.
+ */
+package ch.ethz.globis.pht.pre;
+
+public abstract class ColumnType {
```

Comment:
I removed the ColumnType, the way it was implemented it cause considerable loss of precision. The depth wasn't used at all. See 4f639edec2a990a361620540c750212ec557fd45